homework/basic_html_layout/feedback.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Feedback</title>
+</head>
+
+<body>
+    <h1>My feedback form</h1>
+    <ul>
+        <li>
+            <label>Name:
+                <input type="text" placeholder="Enter your name" required/></label><br>
+        </li>
+        <li>
+            <label>Email:
+                <input type="email" placeholder="Enter your e-mail"/></label><br>
+        </li>
+        <li>
+            <label>Password:
+                <input type="password" placeholder="Enter your password"></label><br>
+        </li>
+        <li>
+            Please check all the emotions that apply to you:
+        </li>
+
+        <ul>
+            <li>
+                <label>Angry
+                    <input type="checkbox" value="checkbox">
```

Comment:
value should be different for each input and should hold input `value`.
`value="checkbox"` does not make any sense and you misunderstood it with `type` attribute.

here as an example value could be `angry`.

Do it everywhere.